spacelift/data_stack.go

```diff
@@ -156,6 +173,12 @@ func dataStackRead(d *schema.ResourceData, meta interface{}) error {
 		d.Set("project_root", nil)
 	}
 
+	if stack.RunnerImage != nil {
+		d.Set("runner_image", *stack.RunnerImage)
```

Comment:
I'm obviously not sure, but wouldn't just d.Set("runner_image", stack.RunnerImage) without the if around it work?

Reply:
You are right, I've applied it to other fields as well.